### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by tsteenbe**
As discussed in TSC meeting of January 28, 2021.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (MarcelBochtler) [inline CODE_OF_CONDUCT.md:63]: I think you missed to add a contact method here.

**Comment by sschuberth** [comment]
Can you please tell in the commit message where the text was taken from?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+For answers to common questions about this code of conduct, see the FAQ at
+[https://www.contributor-covenant.org/faq][FAQ]. Translations are available 
+at [https://www.contributor-covenant.org/translations][translations].
+
+[homepage]: https://www.contributor-covenant.org
+[v2.0]: https://www.contributor-covenant.org/version/2/0/code_of_conduct.html
+[Mozilla CoC]: https://github.com/mozilla/diversity
+[FAQ]: https://www.contributor-covenant.org/faq
+[translations]: https://www.contributor-covenant.org/translations
+
```
Comment (sschuberth) [inline CODE_OF_CONDUCT.md:133]: Nit: Superfluous empty line at the end.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tsteenbe) [reply CODE_OF_CONDUCT.md:63]: @MarcelBochtler  Good catch forgot to save the file after I had insert the email, fixed now

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (tsteenbe) [reply CODE_OF_CONDUCT.md]: Fixed

**Comment by tsteenbe** [comment]
> Can you please tell in the commit message where the text was taken from?

It already in the Attribution section of the file itself but have now also updated the commit message.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**